Whiskey/Tasks/Invoke-WhiskeyNspCheck.ps1

```diff
@@ -113,10 +123,22 @@ function Invoke-WhiskeyNspCheck
 
         Write-Timing -Message 'Running NSP security check'
 
-        $output = Invoke-Command -NoNewScope -ScriptBlock {
-            & $nodePath $nspPath 'check' '--output' 'json' 2>&1 |
-                ForEach-Object { if( $_ -is [Management.Automation.ErrorRecord]) { $_.Exception.Message } else { $_ } }
+        if( !$nspVersion -or $nspVersion -gt (ConvertTo-WhiskeySemanticVersion -InputObject '2.7.0') )

```

Comment:
How about if we use this `if` statement to simply set an `$outputParam` variable to either `--reporter` or `--output`? That way we don't repeat the `Invoke-Command` logic twice.

Reply:
Updated this logic. I added the -ArgumentList parameter to the Invoke-Command so that our Pester tests can see the value being passed. I thought of creating the entire scriptblock outside of the function (so it would resolve the variables for Pester) and passing that in, but chose to go this route..

Reply:
Oh good idea, I hadn't thought of doing it that way.